03/app.js

```diff
@@ -5,13 +5,52 @@ const a = randomNumber(min, max);
 const b = randomNumber(min, max);
 const c = randomNumber(min, max);
 
-console.log(a, b, c);
-
-
-
-
-
-
+console.log('Wylosowano liczby: ' + a, b, c);
+
+const sum = getSum(a, b, c);
+const even = isEven(sum);
+
+console.log('Suma dwóch największych liczb: ' + sum);
+console.log(showInfo(sum, even));
+
+function getSum(a, b, c) {
+    const numberA = parseInt(a);
+    const numberB = parseInt(b);
+    const numberC = parseInt(c);
+    const numbersArr = [numberA, numberB, numberC];
+    numbersArr.sort(function(x, y) {
+        return y - x;
+    });
+    const theLargestNumbers = numbersArr.slice(0, 2);
+    let sumArr = 0;
+    theLargestNumbers.forEach(function(num) {
+        sumArr += num;
+    });
```

Comment:
Skoro wiemy, że 2 największe to dużo prościej będzie napisać `return numbersArr[0] + numbersArr[1]`